### DISCUSSION TITLE
Set Timezone in Octofarm Webinterface

### DISCUSSION THREAD

**Discussion by OXERY** [OctoFarm/OctoFarm#255] [Feature Request]
Like mentioned in issue #254 it would be great to be able to set the server timezone via webinterface, so non-technical users can also profit from the estimated time of the print to finish in their respective timezone.

**Comment by NotExpectedYet** [comment]
Will keep you updated on this. 1.2 will introduce a server setup feature / migration tool so I can define it in there and just update everything as I work through the clean up.
